core-bundle/contao/templates/twig/component/_picture.html.twig

```diff
@@ -30,8 +30,8 @@
             {% set defineProportions = img.width|default(false) and img.height|default(false) %}
             {% set img_attributes = attrs()
                 .set('src', img.src)
-                .set('alt', figure.hasMetadata ? figure.metadata.alt : '')
-                .setIfExists('title', figure.hasMetadata ? (figure.metadata.title ?: null) : null)
+                .set('alt', figure.hasMetadata ? figure.metadata.alt|insert_tag : '')
+                .setIfExists('title', figure.hasMetadata and figure.metadata.title ? figure.metadata.title|insert_tag : null)
```

Comment:
```suggestion
                .setIfExists('title', figure.hasMetadata ? figure.metadata.title|insert_tag : null)
```
Wouldn’t this also work?

Reply:
I don‘t know. Based on the existing `figure.hasMetadata ? (figure.metadata.title ?: null) : null)` I‘d say that `figure.metadata.title` does not have to be defined. So maybe we even have to use `is defined`? @m-vo /cc